NestingContently.Umbraco.ValueConverters/package.xml

```diff
@@ -2,20 +2,21 @@
 <umbPackage>
 	<info>
 		<package>
-			<name>Nesting Contently Value Converters</name>
-			<licence url="https://opensource.org/licenses/MIT">MIT</licence>
-			<url>https://github.com/nathanwoulfe/nestingcontently</url>
+			<name>$title$</name>
+			<version>$version$</version>
+			<licence url="https://opensource.org/licenses/$license$">$license$</licence>
+			<url>$projectUrl$</url>
 			<requirements type="strict">
 				<major>8</major>
 				<minor>7</minor>
 				<patch>0</patch>
 			</requirements>
 		</package>
 		<author>
-			<name>Nathan Woulfe</name>
+			<name>$authors$</name>
 			<website>https://github.com/nathanwoulfe</website>
```

Comment:
Not sure whether `website` is required and/or if we should also move this to the project file.